WordPress/metadata/release_notes.txt

```diff
@@ -1,5 +1,10 @@
-- Added Site Menu and Home tabs to My Site screen
-- Made Add Block button more visible when block editor opens
-- Removed media/audio block upload error text
-- Added Site Intent screen to site creation process
-- Fixed crash caused by notification details for comments on GB File blocks
+* [*] App Settings: updated look and functionality of Locale Picker [https://github.com/wordpress-mobile/WordPress-Android/pull/16331]
+* [*] Fix issue that prevented Jetpack customers with standalone VideoPress subs uploading >5min videos [https://github.com/wordpress-mobile/WordPress-Android/pull/16322]
```

Comment:
Kept this one in the WordPress release notes, as this is about Jetpack-the-service, not Jetpack-the-app, so it's still applicable to the WordPress app too. \cc @leandroalonso